src/modules/host/host-list/host-list.js

```diff
@@ -73,16 +73,6 @@
                 host.status = list[i].status;
                 host.name = list[i].fqdn;
                 host.role = nodeStore.findRole(list[i].tags);
-                host.cluster_name = list[i].tendrlcontext.cluster_name;
```

Comment:
@a2batic HTML contains the cluster Name column, then why you have removed cluster_name property?